src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -21,10 +21,7 @@
 import com.cloudbees.plugins.credentials.common.*;
 import com.cloudbees.plugins.credentials.common.UsernamePasswordCredentials;
 import com.cloudbees.plugins.credentials.domains.DomainRequirement;
-import hudson.Extension;
-import hudson.FilePath;
-import hudson.Launcher;
-import hudson.ProxyConfiguration;
+import hudson.*;
```

Comment:
I don't think wildcards make the code better. There are plenty of tools to keep an up-to-date list of used imports.

Reply:
For me, using wildcard imports or not is a project decision. There are as always advantages and disadvantages. And nowadays tools should be able to handle both.

Reply:
I agree on all counts, I just wanted to make sure it was an intentional change and not somebody's sloppiness that everybody missed.

Reply:
I'm not in general a Java developer and I adopted (ocassionaly, huh) this plugin with worse code conventions than it have today, but there is still no CSC strictly defined here today. We can adopt one, I think, if it make life easier.